concent_api/core/tests/test_unit_authentication_utilities.py

```diff
@@ -108,6 +109,7 @@ def test_validate_golem_message_signed_with_key_should_not_raise_error_if_correc
 
         dumped_ping = dump(ping, CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
         ping = load(dumped_ping, CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
+        ping.sign_message(CONCENT_PRIVATE_KEY)
```

Comment:
please remove the two above lines as they are redundant

Reply:
Done